src/t8_geometry/t8_geometry_implementations/t8_geometry_linear_axis_aligned.hxx

```diff
@@ -121,6 +121,19 @@ struct t8_geometry_linear_axis_aligned: public t8_geometry_with_vertices
     }
     return true;
   }
+
+  void
+  get_tree_bounding_box ([[maybe_unused]] const t8_cmesh_t cmesh, double bounds[6])
+  {
+    T8_ASSERT (cmesh != NULL);
+    T8_ASSERT (active_tree_vertices != NULL);
```

Comment:
A comment would help future devs to understand the computation.

Reply:
Done